fightScreen.py

```diff
@@ -0,0 +1,633 @@
+import sys
+from random import choice as randChoice
+from time import sleep
+from PySide6.QtCore import QSize, Qt, QTimer
+from PySide6.QtGui import QPixmap, QMovie, QColor
+from PySide6.QtWidgets import (
+    QApplication,
+    QMainWindow,
+    QLabel,
+    QWidget,
+    QGridLayout,
+    QSpacerItem,
+    QSizePolicy,
+    QPushButton,
+)
+from meter import Meter
+from arsenal import Arsenal
+
+
+class MainWindow(QMainWindow):
+    def __init__(self):
+        super().__init__()
```

Comment:
Does the `fightScreen` need to be it's own window? I think we should implement something like [here](https://github.com/chanson02/SENG3340/blob/c7290a0b1d6d0a3bde9d5ccf4ca7f98f4a473146/main.py#L16) where the screen gets to live off in the `views` directory and implement it as a `QWidget` subclass, which we can use in `main.py`.